bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -507,4 +456,35 @@ protected Response handleMissingContentQuery( final StoreKey sk, final String pa
         return response;
     }
 
+    private Response redirectContentListing( final String packageType, final String type, final String name,
```

Comment:
This looks similar to the one in the NPM access resource. Would it make sense to consolidate it into something like RequestUtils?